container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/backend_creator/backend_creator.go

```diff
@@ -59,16 +59,17 @@ var (
 func GetDockerKurtosisBackend(
 	optionalApiContainerModeArgs *APIContainerModeArgs,
 	optionalRemoteBackendConfig *configs.KurtosisRemoteBackendConfig,
+	usePodmanMode bool,
```

Comment:
I'm not usually a fan... 
![image](https://github.com/user-attachments/assets/007cfdd5-5ae0-4527-aec7-d8e82d379755)

Reply:
Trueeee lemme use `GetPodmanKurtosisBackend` instead